inference-engine/tests_deprecated/unit/engines/mkldnn/graph/layers/internal/graph_tile_test.cpp

```diff
@@ -215,8 +215,8 @@ class MKLDNNGraphDynBatchTileTests: public MKLDNNGraphTileTests {
             InferenceEngine::CNNNetwork network;
             ASSERT_NO_THROW(network = core.ReadNetwork(model, InferenceEngine::Blob::CPtr()));
 
-            auto implNet = dynamic_cast<InferenceEngine::details::CNNNetworkImpl *>(&((InferenceEngine::ICNNNetwork&)network));
-            ASSERT_NE(nullptr, implNet) << "Failed to cast ICNNNetwork to CNNNetworkImpl";
+            ASSERT_EQ(nullptr, network.getFunction());
+            auto implNet = static_cast<InferenceEngine::details::CNNNetworkImpl *>(&((InferenceEngine::ICNNNetwork&)network));
```

Comment:
why do we need to change it?

Reply:
`network.getFunction()` is an official IE way to check implementation. We can do the same with `dynamic_cast`. Also I have removed virtual from destructor of base class (as we used `std::shared_ptr`) so there is no RTTI with clang.